libp2p/P2PSession.cpp

```diff
@@ -193,13 +216,18 @@ void P2PSession::onTopicMessage(P2PMessage::Ptr message)
                 auto service = m_service.lock();
                 if (service)
                 {
+                    bool versionGt2 = g_BCOSConfig.version() > dev::VERSION::V2_0_0;
                     std::string s = boost::lexical_cast<std::string>(service->topicSeq());
                     for (auto& it : service->topics())
                     {
                         s.append("\t");
-                        s.append(it);
+                        s.append(it.topic);
+                        if (versionGt2)
```

Comment:
```suggestion
                        if (g_BCOSConfig.version() > dev::VERSION::V2_0_0)
```

Reply:
versionGt2 is used in loop condition.so we should get value outside